tools/src/test/scala/com/nvidia/spark/rapids/tool/qualification/QualificationSuite.scala

```diff
@@ -338,6 +341,16 @@ class QualificationSuite extends FunSuite with BeforeAndAfterEach with Logging {
     runQualificationTest(logFiles, "nds_q86_fail_test_expectation.csv")
   }
 
+  test("test event log write format") {
+    val logFiles = Array(s"$logDir/writeformat_eventlog")
+    runQualificationTest(logFiles, "write_format_expectation.csv")
```

Comment:
I would like to see more tests for just the schema parsing section. We can write unit tests that call parseReadSchemaForNestedTypes on a bunch of different schema output... all types from just simple to complext, to nested complext, decimal, etc..

Reply:
Added test to read different schemas. parseReadSchemaForNestedTypes takes in Array[String] so i added tests where we call reportComplexTypes. I hope that's okay. We are testing with eventlogs with no complex types which returns empty, eventlogs with complex and nested complex types which has decimal type in it.